Base.rte/Effects/Pyro/Flame/Flame.lua

```diff
@@ -0,0 +1,52 @@
+function Create(self)
+	self.strength = 80;	-- Damage frequency factor
+	self.checkTimer = Timer();
+	self.checkDelay = math.random(20);
+end
+
+function Update(self)
+	self.ToSettle = false;
+	if self.target and IsMOSRotating(self.target) then
+		self.Vel = self.target.Vel;
+		self.Pos = self.target.Pos - Vector(self.stickPos.X, self.stickPos.Y):RadRotate(self.target.RotAngle - self.tsAngle);
+		local actor = MovableMan:GetMOFromID(self.target.RootID);	-- Inflict damage on the root actor
```

Comment:
I don't think you need to explain that damage is being done to the root actor here. The code makes it fairly clear imo. That said, if you think it's helpful to keep this comment I'm not against it, though I'd rather it be on its own line to make it clear that it relates to lines 12 - 17 rather than just line 12.